tests/unit/amo/components/TestRoutes.js

```diff
@@ -17,14 +17,14 @@ describe(__filename, () => {
     const _config = getFakeConfig({ enableFeatureHomeHeroGuides: false });
     const root = render({ _config });
 
-    expect(root.find(Route)).toHaveLength(30);
+    expect(root.find(Route)).toHaveLength(31);
```

Comment:
Oof. These tests are no bueno. They should only be asserting that the guides route is not available.

Reply:
Yes, you've commented on this test before. I agree it's not great, but I didn't want to change it for this patch.